pkg/trace/api/endpoints.go

```diff
@@ -113,8 +113,8 @@ var endpoints = []endpoint{
 		Handler: func(r *HTTPReceiver) http.Handler { return r.debuggerProxyHandler() },
 	},
 	{
-		Pattern:   "/v0.6/config",
-		Handler:   func(r *HTTPReceiver) http.Handler { return http.HandlerFunc(r.handleConfig) },
+		Pattern:   "/v0.7/config",
```

Comment:
Just two commits ago this was `v0.6`. Why did you change it now?

Reply:
It's breaking (we change the payload format), while we don't have official tracers supporting the older API for now there is some internal stuff using it and your comment reminded me of that

Reply:
It's not necessary, just a nice to have